packages/compass-workspaces/src/components/workspaces.tsx

```diff
@@ -54,6 +54,7 @@ const workspacesContainerStyles = css({
   height: '100%',
   gridTemplateColumns: '100%',
   gridTemplateRows: 'auto 1fr',
+  minWidth: '1024px',
```

Comment:
1024 is probably too wide given that his excludes the sidebar and atlas chrome. Just for demonstration purposes at this stage.

We'll have to deal with the modals separately.